pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -855,6 +858,7 @@ var (
 // KubernetesConfig contains common configuration fields for the control plane components.
 type KubernetesConfig struct {
 	// FeatureGates contains information about enabled feature gates.
+	// This is a legacy field that should no longer be used. Consider using inline map for feature gates definitions.
```

Comment:
I would add an empty line to separate the doc string from the deprecation note.

The comment applies to all occurrences of the doc string.

```suggestion
	//
	// This is a legacy field that should no longer be used. Instead, consider using inline map for feature gates definitions.
```

Reply:
Thanks for the suggestion, added an extra empty line above the deprecation note and included the generated API changes in my latest updates.